packages/google_maps_flutter/example/lib/map_ui.dart

```diff
@@ -182,6 +182,20 @@ class MapUiBodyState extends State<MapUiBody> {
     );
   }
 
+  Widget _locationToggler() {
+    return FlatButton(
+      child:
+          Text('${_options.myLocationEnabled ? 'disable' : 'enable'} location'),
```

Comment:
nit: I'd stay consistent with the Google Maps feature name here and make the label `my location`